models.rdf

```diff
@@ -16,21 +16,10 @@
       <dcterms:publisher rdf:resource="http://www.duraspace.org/"/>
       <rdfs:seeAlso rdf:resource="https://github.com/duraspace/pcdm/wiki"/>
       <rdfs:comment xml:lang="en">Ontology for the Portland Common Data Model, intended to underlie a wide array of repository and DAMS applications.</rdfs:comment>
-      <owl:versionInfo>2015/10/14</owl:versionInfo>
-      <owl:priorVersion rdf:resource="http://pcdm.org/2015/09/28/models"/>
+      <owl:versionInfo>2016/04/18</owl:versionInfo>
+      <owl:priorVersion rdf:resource="http://pcdm.org/2015/10/14/models"/>
```

Comment:
Your update above is correct... and the `versionInfo` date can stay as `2016/04/18`, which is the date of the commit, not the merge.